### ISSUE TITLE
more handsfree approach

### ISSUE THREAD

**Issue by regrett666** [inorichi/tachiyomi#2093]
**App version:** Latest

**Android version:**minimum sdk 

**Issue/Request:** having autoscroll feature

**Steps to reproduce (if applicable)**

 1.can implement autoscroll feature just like other apps? 
 2. 
 3.

**Other details:**
I m new here...so forgive me if i am posting at wrong thread or something

**Comment by hXtreme** [comment]
You might want to edit the title to read: 

[Feature Request] Add Auto-scrolling option to viewer.

Also, you're posting this on the correct thread.

**Comment by Soitora** [comment]
I agree with this feature request, but as @hXtreme said, you need to edit the title (and preferably the content as well) since it's very sloppy.

**Comment by Light-Wing** [comment]
I just wanted to suggest this as well.

the way i think it can be implemented is 
1. have a switch in the reader settings to enable auto scroll
2. if the switch is active then the user can double tap anywhere on the screen to start the auto scroll feature
3. put a speed bar so users can control the auto scroll speed (also can be done as a sub-option under the switch in the reader settings)

hope this gets accepted, I think it will be most useful on Webtoons, to just have it continually scroll down

**Comment by jsmoa921** [comment]
Please make an autscroller,  very useful for devices with big screens such as tablets or people with pain in their hands having to grip and slide at the same tjme

**Comment by LeitHunt** [comment]
Note: 
I searched if these request were already done. 
They were already requested but I wanna present these requests in new perspective :)
[Thanks for such a flawless app & user experience - From a beginner in android development :)]

I have 2 request:
-Pre-set zoom level
-Auto-scroll in vertical mode

This may also help in reduction of battery drain...in amoled screens 
How?:
-Pre-set zoom level:
I have set background full black & always read manga in landscape mode with 50% zoom as by double taping on every page.
So the result is only half screen is covered by a manga page leaving remaining half screen full black.

-Auto scroll: 
Auto scroll request in vertical mode to avoid no. of touches due to double tap for zoom on every page & one tap every time for next page..

Thank you :D

**Comment by Lukyzzz** [comment]
3 years ? Is anyone still working on this ? Was it abandoned ?

**Comment by CrepeTF** [comment]
It's just very very very low priority and there hasn't been anyone interested in it enough to implement it themselves

**Comment by AmineI** [comment]
Using the Camera Switch accessibility feature is a fine workaround. Look left mapped to the swipe left gesture, look right -> swipe right
(or whatever you want really.)
Relevant video from Google https://youtu.be/4fWEfYRlB2A

**Comment by kukaburo** [comment]
Why this isn't implemented already?
